src/main/java/com/ibm/watson/modelmesh/ModelMesh.java

```diff
@@ -3315,6 +3315,7 @@ protected Map<String, ServiceInstanceInfo> getMap(Object[] arr) {
     static final String KNOWN_SIZE_CXT_KEY = "tas.known_size";
     static final String UNBALANCED_KEY = "mmesh.unbalanced";
     static final String DEST_INST_ID_KEY = "tas.dest_iid";
+    static final String VMODELID = "vmodelid";
```

Comment:
nit: Naming consistency 

```suggestion
    static final String VMODEL_ID = "vmodelid";
```

Reply:
I agree with Rafael's nit :-)